### DISCUSSION TITLE
Minimal styles for boilerplates

### DISCUSSION THREAD

**Discussion by carlosbaraza** [blitz-js/blitz#1447] [Ideas]
At the moment, the generated components and pages have no styles at all. They also don't choose any form to style the content. I imagine this is a conscious decision and I like it.

However, I was wondering if it'd be more appealing to some users that want to get started quickly to provide a generator that would make use of some minimal styles to rapidly prototype.

Advantages:
* Getting started faster, as some choices would already be made for the user (e.g. using SASS modules). Generators could bootstrap the styling for the generated files.
* Better onboarding experience, as simple and appealing styles could be more impressive than vanilla unstyled HTML elements.

Disadvantages:
* More boilerplate to be generated
* Some people would not agree with the styling decisions. More opinionated.

Any other ideas?

**Comment by flybayer** [comment]
Yeah, that's a good idea. Hardest part is implementation. I think it may require plugin support, for example where we could add a plugin to override templates for generated code.
